distributed/scheduler.py

```diff
@@ -3965,6 +3965,7 @@ def transition(self, key, finish, *args, **kwargs):
                     self.tasks[ts.key] = ts
                 for plugin in list(self.plugins):
                     try:
+                        kwargs['task_state'] = ts
```

Comment:
Can we move this outside of the for loop?

Reply:
(Sorry for the nitpick)

Reply:
Gentle ping

Reply:
Of course! thanks for the feedback, sorry been away over holiday weekend.